src/app/content/components/ContentLink.tsx

```diff
@@ -50,9 +50,11 @@ export const ContentLink = (props: React.PropsWithChildren<Props>) => {
   return <a
     ref={myForwardedRef}
     onClick={(e) => {
-      if (e.metaKey) {
+      const shouldUseCtrl = navigator && /windows|linux/i.test(navigator.userAgent);
```

Comment:
other routers just bail if any modifiers are pressed, i think we should do the same https://github.com/ReactTraining/react-router/blob/ea44618e68f6a112e48404b2ea0da3e207daf4f0/packages/react-router-dom/modules/Link.js#L18

indeed shift and alt also change the behavior and we would not want to mess with that